frontend/packages/console-app/src/components/cloud-shell/CloudShellExec.tsx

```diff
@@ -252,6 +253,7 @@ const CloudShellExec: React.FC<CloudShellExecProps> = ({
                 setWsReopening(true);
               }
               setWsError(undefined);
+              fireTelemetryEvent('Web Terminal Timeout');
```

Comment:
There's an event that PM wanted to capture which tells us when a terminal times out. Furthermore another event to capture if the user reconnects. They can be the same event, with the latter providing a `reconnect = true` property.

Reply:
have updated it, take a look if it makes sense


```
fireTelemetryEvent('Web Terminal Timeout', {
        reconnect: customResource.status.phase === 'Running',
 });
```

Reply:
Should this be done where it calls `setWsReopening(true)`?

Reply:
yeah, can put it inside as well but kept it above to consider two scenarios.
- Reconnect case (where we do `setWsReopening(true)`) : reconnect to be true i.e `customResource.status.phase === 'Running'` 
- Restart case: : reconnect to be false i.e `customResource.status.phase === 'Running'`